go/saltpack/const.go

```diff
@@ -39,3 +46,22 @@ const SaltPackFormatName = "SaltPack"
 // NoncePrefixEncryption is the prefix used to create the nonce when
 // using the nonce for encryption.
 const NoncePrefixEncryption = "encryption nonce prefix"
+
+// SignatureBlockSize is by default 1MB and can't currently be tweaked.
+const SignatureBlockSize int = 1048576
+
+// SignatureAttachedString is part of the data that is signed in
+// each payload packet.
+const SignatureAttachedString = "attached signature"
+
+// SignatureDetachedString is part of the data that is signed in
+// a detached signature.
+const SignatureDetachedString = "detached signature"
+
+type readState int
+
+const (
+	stateHeader readState = iota
```

Comment:
probably not.

btw, before I refactored this, the encryption states were completely broken...they were all zero.  you set them each to iota.